app/views/users/index.erb

```diff
@@ -0,0 +1,14 @@
+<div class="container">
+
+  <center><h1>Twitter Clone</h1></center>
+  <h4><a href="/users/current_user.id">Profile</a></h4>
+
+  <h3>Users:</h3>
+  <% User.all.each do |user| %>
```

Comment:
In the controller you should do `@users = User.all`. You do not want to do logic in the views. Separate the concerns.